modules/identity-provider-oidc-CR.adoc

```diff
@@ -84,15 +77,17 @@ spec:
         email: <6>
         - custom_email_claim
         - email
+        groups: <7>
+        - groups
       issuer: https://www.idp-issuer.com
 ----
-<1> Optional: Reference to an {product-title} config map containing the
-PEM-encoded certificate authority bundle to use in validating server
-certificates for the configured URL.
-<2> Optional list of scopes to request, in addition to the `openid` scope,
-during the authorization token request.
-<3> Optional map of extra parameters to add to the authorization token request.
-<4> List of claims to use as the preferred user name when provisioning a user
-for this identity. First non-empty claim is used.
-<5> List of claims to use as the display name. First non-empty claim is used.
-<6> List of claims to use as the email address. First non-empty claim is used.
+<1> Optional: Reference to an {product-title} config map containing the PEM-encoded certificate authority bundle to use in validating server certificates for the configured URL.
+<2> Optional: The list of scopes to request, in addition to the `openid` scope, during the authorization token request.
+<3> Optional: A map of extra parameters to add to the authorization token request.
+<4> The list of claims to use as the preferred user name when provisioning a user
+for this identity. The first non-empty claim is used.
+<5> The list of claims to use as the display name. The first non-empty claim is used.
+<6> The list of claims to use as the email address. The first non-empty claim is used.
+<7> The list of claims to use to synchronize groups from the OIDC provider to {product-title} for the user. The first non-empty claim is used.
```

Comment:
@stlaz Is this description okay? Should we work in that it syncs upon login? Like

```
The list of claims to use to synchronize groups from the OIDC provider to {product-title} upon user login.
```

Reply:
I would definitely work the upon login part in, yes :+1:

Reply:
Thanks, updated!